megamek/src/megamek/common/Hex.java

```diff
@@ -590,20 +590,23 @@ public void getUnstuckModifier(int elev, PilotingRollData rollTarget) {
             terrain.getUnstuckModifier(elev, rollTarget);
         }
     }
+    
+    public boolean hasCliffTopTowards(IHex otherHex) {
```

Comment:
Let's put a comment on this public method.

Reply:
Did.